ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -130,6 +130,42 @@
             return this.element.querySelector(Tabs.selectors.qm);
         }
 
+        #syncTabLabels(){
+            var tabs = this._elements["tab"];
+            if (tabs) {
+                if(Array.isArray(tabs)){
+                    tabs.forEach((tab) => {
+                        tab.id = this.getId() + "-tab";
+                        tab.setAttribute("aria-controls", this.getId() + "-tabpanel");
+                    });
+                }else{
+                    tabs.id = this.getId() + "-tab";
+                    tabs.setAttribute("aria-controls", this.getId() + "-tabpanel");
```

Comment:
there seems an inconsistency between tabs and wizard suffix. one is starting with hyphen while the other with underscore.
lets change tabs to use underscore suffix.

Reply:
done